src/c++/perf_analyzer/genai-pa/tests/test_cli.py

```diff
@@ -97,3 +97,10 @@ def test_arguments_model_not_provided(self):
     def test_exception_on_nonzero_exit(self):
         with pytest.raises(GenAiPAException) as e:
             run(["-m", "nonexistent_model"])
+
+    def test_pass_through_args(self):
```

Comment:
How does PA handle duplicate arguments? Which should take priority? There should probably be some tests for a handful of conflicting arguments and making sure they behave however you want it to.

```bash
genai-perf -m my_model --url localhost:8000 -- -u abc:123
genai-perf -m my_model -- -m your_model
...
```